plugins/UFM_NDT_Plugin/ufm_sim_web_service/merger_resources.py

```diff
@@ -181,13 +186,19 @@ def run_ibdiagnet_ndt_compare(self, ndt_file_name):
             if run_ibdiagnet():
                 ibdiagnet_file_path = IBDIAGNET_OUT_NET_DUMP_FILE_PATH
             else:
-                raise ValueError("Report creation failed for %s: Failed to run ibdiagnet" % ndt_file_name)
+                report_content["error"] = "Report creation failed for %s: Failed to run ibdiagnet" % ndt_file_name
```

Comment:
seems like the following 4 lines can be in a function which get the error message as parameter...
you can then reuse this function multiple times during this function...

Reply:
you can not take out raise to function